katz/src/main/kotlin/katz/data/Ior.kt

```diff
@@ -266,17 +266,17 @@ sealed class Ior<out A, out B> {
  *
  * @param f The function to bind across [Ior.Right].
  */
-inline fun <A, B, reified C : A, D> Ior<A, B>.flatMap(aa: Semigroup<C>, crossinline f: (B) -> Ior<C, D>): Ior<C, D> = when (this) {
-    is Ior.Left -> Ior.Left(value as C)
+inline fun <A, B, D> Ior<A, B>.flatMap(AA: Semigroup<A>, crossinline f: (B) -> Ior<A, D>): Ior<A, D> = when (this) {
```

Comment:
If i understood raúl correctly it is a convention to use upper case letter for instances of monads (see https://github.com/FineCinnamon/Katz/pull/31/files#r108131007). As for merging as soon as you approve :D